scripts/test

```diff
@@ -5,5 +5,4 @@ source "${BASH_SOURCE%/*}/../node_modules/sanctuary-scripts/functions"
 
 branches="$(get min-branch-coverage)"
 
-node_modules/.bin/nyc --extension .mjs node_modules/.bin/mocha
-node_modules/.bin/nyc check-coverage --branches $branches
+npx -n='--experimental-modules --no-warnings' oletus
```

Comment:
I switched to a test runner that can load mjs files assuming `--experimental-modules` because the `mocha --require esm` approach had some trouble with nested dependencies.